Content.Server/Labels/Label/Components/LabelComponent.cs

```diff
@@ -1,12 +1,22 @@
 namespace Content.Server.Labels.Components
 {
+    /// <summary>
+    /// Gives opportunity to add labels by entities with the <see cref="HandLabelerComponent"/>
+    /// </summary>
```

Comment:
`Makes entities have a label in their name. Labels are normally given by <see cref="HandLabelerComponent"/>`*